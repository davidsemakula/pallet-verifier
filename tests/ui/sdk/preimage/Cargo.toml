[package]
name = "pallet-preimage"
version = "28.0.0"
#authors.workspace = true
#edition.workspace = true
edition = "2021"
license = "Apache-2.0"
#homepage.workspace = true
#repository.workspace = true
description = "FRAME pallet for storing preimages of hashes"

#[lints]
#workspace = true

[dependencies]
codec = { version = "3.6.12", features = ["derive"], default-features = false, package = "parity-scale-codec" }
scale-info = { version = "2.11.3", default-features = false, features = ["derive"] }
frame-benchmarking = { version = "38.0.0", optional = true, default-features = false }
frame-support = { version = "38.0.0", default-features = false }
frame-system = { version = "38.0.0", default-features = false }
sp-core = { version = "34.0.0", optional = true, default-features = false }
sp-io = { version = "38.0.0", default-features = false }
sp-runtime = { version = "39.0.0", default-features = false }
log = { version = "0.4.22", default-features = false }

[dev-dependencies]
pallet-balances = { version = "39.0.0", default-features = true }
sp-core = { version = "34.0.0", default-features = false }

[features]
default = ["std"]
runtime-benchmarks = [
	"frame-benchmarking",
	"frame-benchmarking/runtime-benchmarks",
	"frame-support/runtime-benchmarks",
	"frame-system/runtime-benchmarks",
	"pallet-balances/runtime-benchmarks",
	"sp-runtime/runtime-benchmarks",
]
std = [
	"codec/std",
	"frame-benchmarking?/std",
	"frame-support/std",
	"frame-system/std",
	"log/std",
	"pallet-balances/std",
	"scale-info/std",
	"sp-core/std",
	"sp-io/std",
	"sp-runtime/std",
]
try-runtime = [
	"frame-support/try-runtime",
	"frame-system/try-runtime",
	"pallet-balances/try-runtime",
	"sp-runtime/try-runtime",
]
