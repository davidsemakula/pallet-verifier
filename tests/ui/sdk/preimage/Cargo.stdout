Searching for FRAME pallet definition ...
Searching for dispatchable function definitions ...
Searching for public associated function definitions ...
Searching for pallet associated function calls ...
Generating tractable entry points for FRAME pallet ...
Analyzing FRAME pallet with MIRAI ...
Creating summaries for FRAME and Substrate functions ...
Analyzing dispatchables ...
Analyzing dispatchable: `ensure_updated` ...
Analyzing dispatchable: `note_preimage` ...
Analyzing dispatchable: `request_preimage` ...
Analyzing dispatchable: `unnote_preimage` ...
Analyzing dispatchable: `unrequest_preimage` ...
